### PR TITLE
[BF-92][BF-96] Implement update news and about us logic

### PR DESCRIPTION
**Pull request by alexkocharyan01**

### CODE REVIEW

File: src/components/AboutUs/AboutUsForm/AboutUsForm.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import React, { useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Translation } from '@constants/translations'
+import { Button } from '@uiComponents/Button'
+import FileField from '@uiComponents/FileField/FileField'
+import TextField from '@uiComponents/FormFields/TextField'
+import Image from 'next/image'
+
+import { useImageUpload } from '../../../hooks/ImageUpload'
```
Comment (ArthurO2002) [inline src/components/AboutUs/AboutUsForm/AboutUsForm.tsx:9]:
```suggestion
import { useImageUpload } from '@hooks/ImageUpload'
```

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/components/AboutUs/AboutUsForm/AboutUsForm.tsx (hunk 1 above)
Comment (alexkocharyan01) [reply src/components/AboutUs/AboutUsForm/AboutUsForm.tsx]: done

File: src/components/AboutUs/AddMember/AboutUsMember.tsx [hunk 2]
```diff
@@ -22,23 +15,11 @@ const defaultValues = {
   header: '',
   paragraph: '',
   imageDescription: '',
-  imageURL: null,
+  imageURL: null ?? '',
```
Comment (ArthurO2002) [inline src/components/AboutUs/AddMember/AboutUsMember.tsx:18]: What does this mean :)
It's always ''

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/components/AboutUs/AddMember/AboutUsMember.tsx (hunk 2 above)
Comment (alexkocharyan01) [reply src/components/AboutUs/AddMember/AboutUsMember.tsx]: done

**Review by ArthurO2002 — APPROVED** [review]

**Pull request merged**